### DISCUSSION TITLE
How to distinguish between a PE installer and a normal executable file

### DISCUSSION THREAD

**Discussion by towatowa** [horsicq/Detect-It-Easy#183] [Q&A] [answered]
How can I tell if it's an installer file or a regular executable program, and where do I look for the code?

**Comment by horsicq** [comment, marked as answer]
Hello! The source code for the whole project: https://github.com/horsicq/DIE-engine
Signatures: https://github.com/horsicq/Detect-It-Easy/tree/master/db
